packages/mdc-textfield/README.md

```diff
@@ -172,21 +172,46 @@ apply these mixins with CSS selectors such as `.foo-text-field.mdc-text-field--i
 
 > _NOTE_: the `mdc-line-ripple-color` mixin should be applied from the not focused class `foo-text-field:not(.mdc-text-field--focused)`).
 
+#### Mixins for Text Field Box
+
+Mixin | Description
+--- | ---
+`mdc-text-field-box-corner-radius($radius)` | Customizes the border radius for the text field box variant.
+`mdc-text-field-box-fill-color($color)` | Customizes the background color of the text field box.
+
+#### Mixins for Text Field Outline
+
+Mixin | Description
+--- | ---
+`mdc-text-field-outline-color($color)` | Customizes the border color of the outline text field.
+`mdc-text-field-outlined-corner-radius($radius)` | Sets the border radius of of the text field outline variant.
```

Comment:
@bonniezhou I think this should be changed to `mdc-text-field-outline-corner-radius` what do you think?

Reply:
Actually I'm going to change `mdc-text-field-outline-color` --> `mdc-text-field-outlined-color` so it isn't a breaking change.

Reply:
How is that not a breaking change? AFAICT `mdc-text-field-outline-color` is also [already a thing that exists](https://github.com/material-components/material-components-web/blob/master/packages/mdc-textfield/outline/_mixins.scss#L19).

Plus, it seems like the one "outlined" name is the outlier, and "outline" sounds more natural to me in mixin names, so IMO I'd go with your first suggestion.

Reply:
True, but that `mdc-text-field-outline-color` is in the mdc-text-field__outline package, which is moved to the mdc-notched-outline. I this would cover it:

```
BREAKING CHANGE: removed mdc-text-field__outline element for mdc-notched-outline```

But yes you're right, `mdc-text-field-outline-color` and `mdc-text-field-outline-corner-radius` are more natural. It also matches the variant name and is thus less confusing. I'll go with that.

Reply:
Yeah I think initially "Outlined" should refer to the variant of Text Field and "Outline" refers to the actual subcomponent. But changing them both to "Outline" sounds like a good idea, and would cause less confusion.